samples/ResourceOwnerPasswordFlow/Providers/AuthorizationProvider.cs

```diff
@@ -0,0 +1,65 @@
+﻿
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using AspNet.Security.OpenIdConnect.Server;
+using AspNet.Security.OpenIdConnect.Extensions;
+using Microsoft.Framework.DependencyInjection;
+using System.Security.Claims;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.AspNet.Hosting;
+
+namespace ResourceOwnerPasswordFlow.Providers
+{
+    public sealed class AuthorizationProvider : OpenIdConnectServerProvider
+    {
+        private UserManager<IdentityUser> _userManager;
+
+        public AuthorizationProvider(UserManager<IdentityUser> userManager)
```

Comment:
Try to resolve the user manager directly in `GrantResourceOwnerCredentials` using `notification.HttpContext.RequestServices`.